### PR TITLE
ENH: update to ITK 5.3.0

### PR DESCRIPTION
**Pull request by dyollb**
The currently released version 1.2.0 is not compatible with ITK 5.3.0:

image

This PR adds a dependency to itk-5.3.0 for the python package. The PR also updates the github actions to the latest template.

Fixes issue #37

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,7 +50,5 @@
     license='Apache',
     keywords='ITK InsightToolkit Math-morphology Label-images',
     url=r'https://github.com/InsightSoftwareConsortium/ITKLabelErodeDilate',
-    install_requires=[
-        r'itk>=5.3rc4'
-    ]
-    )
+    install_requires=[r'itk~=5.3.0']
```
Comment (dzenanz) [inline setup.py:53]: What does `~=` mean here? Why not keep `>=`?

File: .github/workflows/build-test-package.yml [hunk 2]
```diff
@@ -3,276 +3,10 @@ name: Build, test, package
...
-      - name: Publish 🐍 Python 📦 package to PyPI
-        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
-        uses: pypa/gh-action-pypi-publish@master
-        with:
-          user: __token__
-          password: ${{ secrets.pypi_password }}
+  python-build-workflow:
+    uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@5083da2740617b78423ebf6083489e1e70ee8ca0
+    secrets:
+      pypi_password: ${{ secrets.pypi_password }}
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml:12]: Add a new line at the end of file for better unix compatibility.

**Review by dzenanz — COMMENTED** [review]
Thanks for working on this. Let's see what the CI says.

File: setup.py (hunk 1 above)
Comment (dyollb) [reply setup.py:53]: https://stackoverflow.com/questions/39590187/in-requirements-txt-what-does-tilde-equals-mean/39590286#39590286

File: .github/workflows/build-test-package.yml (hunk 2 above)
Comment (dyollb) [reply .github/workflows/build-test-package.yml]: Fixed

**Review by dzenanz — APPROVED** [review]
To be squashed upon merge.

**Comment by dyollb** [comment]
> To be squashed upon merge.

Also needs to be tagged (v1.2.1) to trigger publishing to pypi. Thanks

**Pull request merged**